Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG NIM_COMMIT
 ARG LOG_LEVEL=TRACE
 
 # Get build tools and required header files
-RUN apk add --no-cache bash git build-base pcre-dev linux-headers curl rust cargo
+RUN apk add --no-cache bash git build-base pcre-dev linux-headers curl jq rust cargo
```

Comment:
an FYI since I had this issue. When using the images built on quay (per PR and latest master) [see](https://quay.io/repository/wakuorg/nwaku-pr?tab=tags) these wont be applied since its using another file:

https://github.com/waku-org/nwaku/blob/master/docker/binaries/Dockerfile.bn.amd64#L16

So perhaps we should add it there as well (depending on the use case)

Not sure I like having multiple comose files, but thats another story.

Reply:
No, look at what that `Dockerfile` does:
https://github.com/waku-org/nwaku/blob/11e7394e0d767e3d7aa570c92d404c2c6e03bfce/docker/binaries/Dockerfile.bn.amd64#L22-L23
It copies the binary, it doesn't build it.